keyboards/redox/keymaps/jeherve/keymap.c

```diff
@@ -25,31 +25,38 @@ enum custom_keycodes {
 	WAVE,
 	YOSHI,
 	THUMB_UP,
-	NBSP
+	NBSP,
+  INVOKE_1P,
```

Comment:
You may want to try to keep thins to 7 characters long.  That's the standard length, and is what will fit better "visually". 

This isn't a requirement, just a suggesion.

Reply:
Makes sense. Done in 6bd230a1d19f976100a846e9d2c9fa7413531089